packages/@aws-cdk/aws-lambda-nodejs/lib/bundling.ts

```diff
@@ -169,13 +170,9 @@ export class Bundling implements cdk.BundlingOptions {
         }
         tsconfig = path.relative(this.projectRoot, findConfig);
       }
-
+      compilerOptionsString = tryGetTsconfigCompilerOptions(pathJoin(options.inputDir, tsconfig));
+      tscCommand = `${options.tscRunner} ${relativeEntryPath} ${compilerOptionsString}`;
```

Comment:
Should the entry path be also enclosed with double quotes (") here to avoid issues with spaces?

Reply:
Indeed